homeassistant/components/trafikverket_train/sensor.py

```diff
@@ -129,12 +130,13 @@ def __init__(self, train_api, name, from_station, to_station, weekday, time):
         self._state = None
         self._departure_state = None
         self._delay_in_minutes = None
+        self._timezone = timezone("Europe/Stockholm")
```

Comment:
We have this util that returns a tzinfo object that can be used with `datetime.astimezone`.

https://github.com/home-assistant/core/blob/5e25df91b232798cb43a08cc9baa27edfa0b286e/homeassistant/util/dt.py#L45-L46

https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone

We don't use pytz anymore.